init.lua

```diff
@@ -82,6 +105,7 @@ local function preview()
 
    -- create a list that holds the clients to preview, from left to right
    local leftRightTab = {}
+   local leftRightTabToAltTabIndex = {} -- save mapping from leftRightTab to altTabTable as well
```

Comment:
> as you already did a few lines below with the event handler.

I don't get this one. I'm using the table for the event handler. So without the table, the whole implementation would have to be changed (see my first comment some days ago). I don't see a problem with an additional table which only holds some integers?